product_docs/docs/migration_portal/3.4.0/05_mp_advanced_data_migration.mdx

```diff
@@ -9,12 +9,9 @@ legacyRedirectsGenerated:
 
 <div id="mp_advanced_data_migration" class="registered_link"></div>
 
-For larger databases that require a parallel data load, you can use one of the following methods:
+For larger databases that require a parallel data load, you can use one of the following:
```

Comment:
Feels like the word "methods" added some value in the lead in sentence.

Reply:
I think it's fine either way but I added it back,